### ISSUE TITLE
Rule for Doze Emote Issue

### ISSUE THREAD

**Issue by AsunaTsuki** [NightmareXIV/DynamicBridge#18]
When i set a rule to activate on doze (the one where you lay down in bed) will work for the first few seconds as the emote is starting but after that, it untriggers and then retriggers before going back to untriggering once the emote finishes activating.

**Comment by PrincessRTFM** [comment]
I have a similar issue, but it doesn't untrigger at the end. It applies as I lie down, untriggers for a _split_ second, retriggers, and stays active. However, if I then change poses, going from the "lying on side" pose (female miqo'te, I don't know the number) to the "lying on belly" pose briefly untriggers and then retriggers the same way.

**Comment by shadowking1443** [comment]
Follow up to this, I encountered the same issue and believe it is related to how the game handles "Doze" and sleeping. DB will activate for the first few seconds upon using the doze emotion, but if you check the emotion section of DB once laying down, it will list the current emote as "Unknown". There appears to be 3-4 of these unknown emotes depending on the sleep pose. Adding all of them to the rule will allow it to apply during all doze positions.

**Comment by Limiana** [comment]
Yeah, ideally I need to find all combos and group them up together under the same emote name. Though, maybe some people will want to use them separately too
